tests/cli/test_flow_run.py

```diff
@@ -587,3 +587,69 @@ async def test_when_num_logs_passed_with_reverse_param_and_num_logs(
             ],
             expected_line_count=10,
         )
+
+    async def test_passing_head_and_tail_raises(self, flow_run_factory):
+        # Given
+        flow_run = await flow_run_factory(num_logs=self.LOGS_DEFAULT_PAGE_SIZE + 1)
+
+        # When/Then
+        await run_sync_in_worker_thread(
+            invoke_and_assert,
+            command=[
+                "flow-run",
+                "logs",
+                str(flow_run.id),
+                "--tail",
+                "--num-logs",
+                "10",
+                "--head",
+            ],
+            expected_code=1,
+            expected_output_contains=(
+                "Please provide either a `head` or `tail` option but not both."
+            ),
+        )
+
+    async def test_default_tail_returns_default_num_logs(self, flow_run_factory):
+        # Given
+        flow_run = await flow_run_factory(num_logs=self.LOGS_DEFAULT_PAGE_SIZE + 1)
+
+        # When/Then
+        await run_sync_in_worker_thread(
+            invoke_and_assert,
+            command=["flow-run", "logs", str(flow_run.id), "-t"],
+            expected_code=0,
+            expected_output_contains=[
+                f"Flow run '{flow_run.name}' - Log {i} from flow_run {flow_run.id}."
+                for i in range(
+                    self.LOGS_DEFAULT_PAGE_SIZE - 9, self.LOGS_DEFAULT_PAGE_SIZE
+                )
+            ],
+            expected_line_count=20,
+        )
+
+    async def test_reverse_tail_with_num_logs(self, flow_run_factory):
```

Comment:
Ah, could we add 2 more tests: 
- one called `test_reverse_tail_returns_default_num_logs` (in TIMESTAMP DESC order)
- and one called `test_when_num_logs_greater_than_page_size_with_tail_outputs_correct_num_logs` (in TIMESTAMP ASC order)